src/Uno.UI/UI/Xaml/Media/RectangleGeometry.wasm.cs

```diff
@@ -0,0 +1,36 @@
+﻿#nullable enable
+using Uno.Extensions;
+using Uno.UI.DataBinding;
+using Windows.UI.Xaml.Wasm;
+
+namespace Windows.UI.Xaml.Media
+{
+	partial class RectangleGeometry
+	{
+		private readonly SvgElement _svgElement = new SvgElement("rect");
```

Comment:
`RectangleGeometry` can also be used for `UIElement.Clip` property. Correct me if I'm wrong, but I think that it means that we are creating a `SvgElement` (which is a `UIElement`, i.e. with a lot of interop at creation) for nothing when used for clipping. I think that this `_svgElement` should be lazy.

Reply:
Makes sense. Do you see a problem with any of the others, or just `RectangleGeometry`?